source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/skin.html

```
@@ -1,79 +1,77 @@
+    <hr>
+      <li class='caMenuItem debugging'><?tr("Debugging");?></li>
+    </ul>
   </div>
-</span>
\ No newline at end of file
+  </span>
+  <div class='mainArea'>
+  <div class='spinnerBackground'></div>
+  <div class='spinner fixed'></div>
+  <strong><font size='3'></strong></font>
+  <div class='searchArea onlyShowWithFeed'><span id='searchFilter'><input type='text' id='searchBox' placeholder='<?tr('Search');?>'><span class='searchSubmit' onclick='doSearch(true)'><i id='searchButton' class='fa fa-search fa-lg'></i></span><span class='caButton maxPerPage' id='maxPerPage' style='display:none;'></span></div>
```

HTMLHint (1.5.0)

[error] The value of attribute [ class ] must be in double quotes.

(attr-value-double-quotes)